src/test/java/ch/wisv/areafiftylan/IntegrationTest.java

```diff
@@ -127,6 +125,21 @@ public void tearDownIntegrationTest() {
         RestAssured.reset();
     }
 
+
+    protected Header getCSRFHeader() {
```

Comment:
Don't you want to change the authentication in the tests already here? CSRF is going to be disabled at some point in time.